api/src/main/java/org/motechproject/nms/api/web/contract/mobileAcademy/sms/DeliveryInfo.java

```diff
@@ -36,13 +36,13 @@ public void setDeliveryStatus(DeliveryStatus deliveryStatus) {
     }
 
     private static String obscure(String a) {
-        return "tel: *******" + a.substring(a.length() - MASK_LENGTH);
+        return a == null ? "null" : "tel: *******" + a.substring(a.length() - MASK_LENGTH);
     }
 
     @Override
     public String toString() {
         return "DeliveryInfo{" +
-                "address='" + obscure(address) + '\'' +
+                "address='" +  obscure(address) + '\'' +
```

Comment:
I originally had the null check there, but then figured it would be better
in the obscure method.  I guess an extra ' ' didn't want to be deleted.
Poor little guy, can we keep him?

On Mon, Aug 3, 2015 at 5:55 PM, Frank Huster <notifications@github.com>
wrote:

> In
> api/src/main/java/org/motechproject/nms/api/web/contract/mobileAcademy/sms/DeliveryInfo.java
> <https://github.com/motech-implementations/mim/pull/555#discussion_r36148337>
> :
>
> >      }
> >
> >      @Override
> >      public String toString() {
> >          return "DeliveryInfo{" +
> > -                "address='" + obscure(address) + '\'' +
> > +                "address='" +  obscure(address) + '\'' +
>
> Double space?!? OMG YOU'RE CRAZY!!!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/555/files#r36148337>.
>